butane_core/src/migrations/adb.rs

```diff
@@ -351,6 +351,38 @@ impl DeferredSqlType {
         }
     }
 }
+/// Compare, with Known and KnownId being identical if they contain the same type.
+impl PartialEq<DeferredSqlType> for DeferredSqlType {
+    fn eq(&self, other: &DeferredSqlType) -> bool {
+        match self {
```

Comment:
I think there's a more succinct formulation of this match statement. See https://github.com/Electron100/butane/commit/3a1e07cafcd6d2e580105551be772afb17034660 if you'd like to adopt it